client-http/src/test/scala/com/cloudera/livy/client/http/HttpClientSpec.scala

```diff
@@ -18,11 +18,15 @@
 
 package com.cloudera.livy.client.http
 
+import java.io.{ByteArrayInputStream, File}
 import java.net.{InetAddress, URI}
+import java.nio.file.{Paths, Files}
```

Comment:
I think you are looking at an outdated diff. The style checker is running - here is one that failed the style check: https://travis-ci.org/cloudera/livy/builds/106647672

Travis results is a bit confusing (it runs `mvn install -DskipTests` and then `mvn clean package` - so the install run's logs are not visible unless the install run fails fail)